plugins/inputs/marklogic/README.md

```diff
@@ -0,0 +1,50 @@
+# Marklogic Plugin
+
+The Marklogic Telegraf plugin gathers status metrics from one or more host in the Marklogic cluster.
+
+### Configuration:
+
+```toml
+# Description
+[[inputs.marklogic]]
+
+## List URLs of Marklogic hosts using Management API endpoint.
+# hosts = ["http://localhost:8002/manage/v2/hosts/${hostname}?view=status&format=json"]
+
+# Using HTTP Digest Authentication.
+# digest_username = "telegraf"
+# digest_password = "p@ssw0rd"
+```
+
+### Measurements & Fields:
```

Comment:
Follow the pattern [in the github plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/github#metrics) for this heading.